### DISCUSSION TITLE
我想购买专业版本，使用支付宝支付，但是提示商户异常，无法付款

### DISCUSSION THREAD

**Discussion by moneymuchmuch** [getlantern/lantern-forum-cn#157] [Q&A]
我想购买专业版本，使用支付宝支付，但是提示商户异常，无法付款

**Comment by MaxMKMax** [comment]
@ladyssofy 您好。我跟支付部门跟进了。如果您没有付款选项，可以采用这个方式。
您可以用这个链接支付。谢谢您的支持。非常开心有您这么尊贵的用户。

https://buy.stripe.com/eVacPZe3A5cp1oc4gO
